src/main/java/org/launchcode/walkabout/models/dto/UserJournal.java

```diff
@@ -1,12 +1,24 @@
-package src.main.java.org.launchcode.walkabout.models.dto;
-import java.time.LocalDate;
+package org.launchcode.walkabout.models.dto;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
 import java.time.LocalDateTime;
 import java.util.Scanner;
 
-public class UserJournal {
+
+public class UserJournal{
+
+    @NotBlank(message = "Please add the journal entry.")
+    @Min(0)
     private String journalEntry;
+
+    @NotBlank(message = "Please add the journal location.")
+    @Min(0)
```

Comment:
Since this looks like a required field, should we make min to be at least 1 character (or maybe 3)? I feel like most abbreviations of places would likely be 3 (e.g. St. Louis - STL).